sdk/ai/azopenai/ci.yml

```diff
@@ -35,6 +35,7 @@ extends:
       UsePipelineProxy: false
       CloudConfig:
         Public:
+          ServiceConnection: azure-sdk-tests-openai
           SubscriptionConfigurations:
```

Comment:
Checking a couple of them they seem to be duplicated in the sub-config and as individual secrets.

Reply:
I know we're trying to remove the sub config for base auth stuff, but the model of dumping individual secrets into one mega keyvault, and enumerating them individually in yaml, is the one we originally wanted to get away from when we implemented the sub config. I didn't realize that python and js were still following that, and I explicitly asked the go team to not do that during openai live test onboarding.

Reply:
I would prefer we go away from sub-configs in KV completely. Especially if they contain multiple secrets because it makes updating those secrets all the more difficult.

Reply:
Let's continue to discuss offline, I agree we need an improved solution. I'd like to avoid all the annoyances we had with the earlier approach (refactoring enumerated variable groups and secret names across yaml files*repos, values being stored in duplicate, local test limitations, access control limitations, etc.).